### PR TITLE
cloud: Add `ticloud cluster connect-info` doc

### PR DESCRIPTION
**Pull request by Daemonxiao**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add `ticloud cluster connect-info` doc
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/ticloud-cluster-connect-info.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+| -h, --help                 | Help information for this command                                                                                                                                                                                                                                                                                                                                          | No       | Works in both non-interactive and interactive modes. |
+| -o, --output string        | Output format (`human` by default). Valid values are `human` or `json`. To get a complete result, use the `json` format.                                                                                                                                                                                                                                                   | No       | Works in both non-interactive and interactive modes. |
```
[the remark this answers is not in the document]
Comment (Daemonxiao) [reply tidb-cloud/ticloud-cluster-connect-info.md]: @zhangyangyu Thanks for your review. I have deleted the `--output` flag.

**Review by zhangyangyu — APPROVED** [review]

File: tidb-cloud/ticloud-cluster-connect-info.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+| -c, --cluster-id string    | The ID of the cluster                                                                                                                                                                                                                                                                                                                                                      | Yes      | Only works in non-interactive mode.                  |
+| --client string            | The desired client used for the connection. Supported clients include `general`, `mysql_cli`, `mycli`, `libmysqlclient`, `python_mysqlclient`, `pymysql`, `mysql_connector_python`, `mysql_connector_java`, `go_mysql_driver`, `node_mysql2`, `ruby_mysql2`, `php_mysqli`, `rust_mysql`, `mybatis`, `hibernate`, `spring_boot`, `gorm`, `prisma`, `sequelize_mysql2`, `django_tidb`, `sqlalchemy_mysqlclient`,  and `active_record`. | Yes      | Only works in non-interactive mode.                  |
```
Comment (qiancai) [inline tidb-cloud/ticloud-cluster-connect-info.md:36]:
```suggestion
| --client string            | The desired client used for the connection. Supported clients include `general`, `mysql_cli`, `mycli`, `libmysqlclient`, `python_mysqlclient`, `pymysql`, `mysql_connector_python`, `mysql_connector_java`, `go_mysql_driver`, `node_mysql2`, `ruby_mysql2`, `php_mysqli`, `rust_mysql`, `mybatis`, `hibernate`, `spring_boot`, `gorm`, `prisma`, `sequelize_mysql2`, `django_tidb`, `sqlalchemy_mysqlclient`, and `active_record`. | Yes      | Only works in non-interactive mode.                  |
```

**Review by hfxsd — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

**Pull request merged**